common-ui/src/main/res/drawable/ic_platform_macos_16.xml

```diff
@@ -0,0 +1,19 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="16dp"
+    android:height="16dp"
+    android:viewportWidth="16"
+    android:viewportHeight="16">
+  <path
+      android:pathData="M5,4.75C5,4.336 4.664,4 4.25,4C3.836,4 3.5,4.336 3.5,4.75V6.25C3.5,6.664 3.836,7 4.25,7C4.664,7 5,6.664 5,6.25V4.75Z"
+      android:fillColor="#000000"
+      android:fillAlpha="0.84"/>
```

Comment:
When adding icons, you need to remove alpha and replace color to `?attr/daxColorPrimaryIcon` so they look good in light/dark mode
```suggestion
      android:fillColor="?attr/daxColorPrimaryIcon" />
```